tests/foreman/ui/test_contentview.py

```diff
@@ -1289,8 +1382,9 @@ def test_positive_publish_with_custom_content(self):
             self.assertIsNotNone(self.content_views.wait_until_element(
                 common_locators['alert.success_sub_form']))
 
-    @stubbed()
+    @run_in_one_thread
     @run_only_on('sat')
+    @skip_if_not_set('fake_manifest')
     @tier2
     def test_positive_publish_composite_with_custom_content(self):
```

Comment:
Do we really need this separate test for publish? Because I think Promote test already taking care of publish.
So can we remove this in favor of objective of this testcase is been covered in promote test case ?